package.json

```
@@ -7,7 +7,7 @@
     "start": "webpack -w",
     "build": "webpack",
     "test": "jest",
-    "lint": "eslint . --ext .ts"
+    "lint": "eslint . --ext .ts && stylelint 'src/**/*.scss'"
```

test

---

It would go like this:
```suggestion
    "lint": "eslint . --ext .ts"
    "lint:scss": "stylelint 'src/**/*.scss'"
```